src/TuDa.CIMS.Web/Components/Pages/WorkingGroupListPage.razor.cs

```diff
@@ -15,6 +15,8 @@ public partial class WorkingGroupListPage
     [Inject]
     private IWorkingGroupApi _workingGroupApi { get; set; } = null!;
 
+    private WorkingGroupPageWorkingGroupList? _workingGroupList;
```

Comment:
🔧 This should not be nullable. When using `@ref` the component is never null. You can suppress the warning like this.
```suggestion
    private WorkingGroupPageWorkingGroupList _workingGroupList = null!;
```